### ISSUE TITLE
Handle changes to brew install method (--upgrade no longer supported)

### ISSUE THREAD

**Issue by IanWhalen** [mongodb/server-workflow-tool#3]
maybe you could also add a "skip" option here?  typing 'atom' or 'sublime-text' seems to fail:

```
brew install --upgrade sublime-text
Encountered a bad command exit code!

Command: 'brew install --upgrade sublime-text'

Exit code: 1

Stdout:



Stderr:

                                     end of the run.
    -i, --interactive                Download and patch formula, then open a
                                     shell. This allows the user to run
                                     ./configure --help and otherwise
                                     determine how to turn the software package
                                     into a Homebrew package.
    -g, --git                        Create a Git repository, useful for
                                     creating patches to the software.
    -h, --help                       Show this message.
Error: invalid option: --upgrade
```

**Comment by IanWhalen** [comment]
So yeah, we need to fix handling of --upgrade here, but should also make this able to skip maybe?

**Comment by IanWhalen** [comment]
Fixed in fd3f596fef52c85d20679eb3cf55b4dc2172e96d
